client/packages/programs/src/JsonForms/components/IdGenerator.tsx

```diff
@@ -186,10 +186,10 @@ const valueFromPart = async (
   switch (part.type) {
     case 'Field': {
       const field = extractProperty(data, part.field);
-      if (field === undefined || typeof field !== 'string') {
+      if (field === undefined) {
         return undefined;
       }
-      return field;
+      return String(field);
```

Comment:
Is this actually the same as \`${field}\`?

Reply:
As far as I can tell, yes.